datarepo-clienttests/src/main/java/utils/DataRepoUtils.java

```diff
@@ -48,6 +51,30 @@ public static JobModel waitForJobToFinish(RepositoryApi repositoryApi, JobModel
     return job;
   }
 
+  /**
+   * Poll for running job. Polls for designated time.
+   *
+   * @param repositoryApi the api object to use
+   * @param job the job model to poll
+   * @param pollTime time in seconds for the job to poll before returning
+   */
+  public static JobModel pollForRunningJob(RepositoryApi repositoryApi, JobModel job, int pollTime)
```

Comment:
I like your new method that allows overriding the timeout. Perhaps make the waitForJobToFinish method just call your new one with the maximumSecondsToWaitForJob property, so we don't have too much duplicate code?

Reply:
Oh, I like that! Thanks!